Gopkg.toml

```diff
@@ -72,6 +72,11 @@
 ## Some repos dont have releases.
 ## Pin to revision
 
+[[constraint]]
+  name = "golang.org/x/crypto"
```

Comment:
not a fan of doing this. if I am a developer and I see an import "github.com/.../...", I usually go to github.com to check out the source code for that import. But here **you get what you see** concept is broken, so I had to be aware of the alias ??somehow??

Reply:
Ok - would a comment everywhere we import this that it's actually an alias be sufficient?

Reply:
Yep. Or we could indicate that it's a fork by appending `fork` or `tendermint` to name:`tendermint.com/golang.org/x/crypto` ||`golang.tendermint.com/x/crypto`

Reply:
Hmm - I like this idea but `dep` doesn't want to let me do it so long as these URLs don't exist. Any ideas? Or do we need to create the valid endpoint first?

Reply:
> I like this idea but dep doesn't want to let me do it so long as these URLs don't exist

did not know that! probably not worth the efforts short-term

Reply:
Golang forking voodoo. It breaks things like `internal` packages

Reply:
ohh, ok this lgtm then